---
layout: ont
id: clo
contact: 
  email: Sirarat.Sarntivijai@fda.hhs.gov
  label: Sirarat Sarntivijai
description: The Cell Line Ontology (CLO) is a community-driven ontology that is developed to standardize and integrate cell line information and support computer-assisted reasoning.
homepage: http://www.clo-ontology.org
products: 
  - id: clo.owl
title: Cell Line Ontology
contact: 
  email: huang@southalabama.edu
  label: Huang, Jingshan
homepage: http://omit.cis.usouthal.edu/
id: omit
page: http://omit.cis.usouthal.edu/
products: 
  - id: omit.owl
title: Ontology for MIRNA Target Prediction
contact: 
  email: Terry.Hayamizu@jax.org
  label: Terry Hayamizu
description: A structured controlled vocabulary of the adult anatomy of the mouse (Mus).
domain: anatomy
homepage: http://www.informatics.jax.org/searches/AMA_form.shtml
id: ma
products: 
  - id: ma.owl
taxon: 
  id: NCBITaxon:10088
  label: Mus
title: Mouse adult gross anatomy
tracker: https://sourceforge.net/tracker/?group_id=76834&atid=1205376
contact: 
  email: burkesquires@gmail.com
  label: Burke Squires
domain: health
homepage: http://purl.obolibrary.org/obo/flu/
id: flu
products: 
  - id: flu.owl
title: Influenza Ontology
tracker: http://purl.obolibrary.org/obo/flu/tracker
contact: 
  email: Terry.Hayamizu@jax.org
  label: Terry Hayamizu
description: A structured controlled vocabulary of abstract (i.e. non-stage specific) stage-specific anatomical structures of the mouse (Mus).
domain: anatomy
homepage: http://emouseatlas.org
id: emapa
page: http://www.obofoundry.org/wiki/index.php/EMAPA:Main_Page
products: 
  - id: emapa.owl
taxon: 
  id: NCBITaxon:10088
  label: Mus
title: Mouse gross anatomy and development, abstract
contact: 
  email: yongqunh@med.umich.edu
  label: Yongqunh He
description: To ensure consistent representation of vaccine knowledge and to support automated reasoning, a community-based effort to develop the Vaccine Ontology (VO) has been initiated. This international collaborative VO development effort has been joined by many institutes and disciplines around the world. The Vaccine Ontology is also being used in a variety of applications by many user groups in academia and industry.
domain: health
homepage: http://www.violinet.org/vaccineontology
id: vo
products: 
  - id: vo.owl
title: Vaccine ontology
tracker: http://sourceforge.net/tracker/?atid=1127137&group_id=255029&func=browse
contact: 
  email: hastings@ebi.ac.uk
  label: Janna Hastings
description: "FIX consists of two ontologies: methods and properties (but not objects, which are subject of the chemical ontology). The methods are applied to study the properties."
id: fix
products: 
  - id: fix.owl
title: Physico-chemical methods and properties
contact: 
  email: djs93@gen.cam.ac.uk
  label: David Osumi-Sutherland
description: A structured controlled vocabulary of the development of <i>Drosophila melanogaster</i>.
domain: anatomy
homepage: http://www.flybase.org/
id: fbdv
products: 
  - id: fbdv.owl
taxon: 
  id: NCBITaxon:7227
  label: Drosophila
title: Drosophila development
tracker: https://cedar.bio.indiana.edu/trac/env/ontologies/wiki/PublicWikiStart
contact: 
  email: yongqunh@med.umich.edu
  label: Yongqun Oliver He
description: "The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology. "
homepage: http://ogg.googlecode.com/
id: ogg
products: 
  - id: ogg.owl
title: The Ontology of Genes and Genomes
tracker: https://code.google.com/p/ogg/issues/list
contact: 
  email: geg18@aber.ac.uk
  label: George Gkoutos
description: Metrical units for use in conjunction with PATO
domain: phenotype
homepage: http://code.google.com/p/unit-ontology/
id: uo
products: 
  - id: uo.owl
title: Units of measurement
tracker: http://sourceforge.net/tracker/?group_id=76834&atid=595654
contact: 
  email: bakerc@unb.ca
  label: Christipher Baker
description: "Lipid research is increasingly integrated within systems level biology such as lipidomics where lipid classification is required before appropriate annotation of chemical functions can be applied. The ontology describes the LIPIDMAPS nomenclature classification explicitly using description logics (OWL-DL). Lipid classes are organized hierarchically with the super-classes restricted by generic necessary conditions. More specific necessary conditions are used to define membership requirements for sub classes of lipid according to appropriate functional groups. "
domain: lipids
id: lipro
products: 
  - id: lipro.owl
title: Lipid Ontology
contact: 
  email: rwalls2008@gmail.com
  label: Ramona Walls
description: The Biological Collections Ontology (BCO) is a being developed as an application ontology as part of the Biocode Commons project, within the OBO Foundry framework. The goal of the BCO is to support the interoperability of biodiversity data, including data on museum collections, environmental/metagenomic samples, and ecological surveys.
homepage: http://code.google.com/p/bco/
id: bco
products: 
  - id: bco.owl
title: Biological Collections Ontology
tracker: http://code.google.com/p/bco/issues/list
contact: 
  email: zfinadmin@zfin.org
  label: ZFIN administrators
description: A developmental stages of the Zebrafish (<i>Danio rerio</i>). Note that ZFA includes the leaf nodes of this ontology
domain: anatomy
homepage: http://zfin.org/zf_info/anatomy/dict/sum.html
id: zfs
page: http://code.google.com/p/developmental-stage-ontologies
products: 
  - id: zfs.owl
taxon: 
  id: NCBITaxon:7954
  label: Danio
title: Zebrafish developmental stages
contact: 
  email: mah79@cam.ac.uk
  label: Midori Harris
description: FYPO is a formal ontology of phenotypes observed in fission yeast. FYPO is being developed to support the comprehensive and detailed representation of phenotypes in PomBase, the new online fission yeast resource (www.pombase.org). Its scope is similar to that of the Ascomycete Phenotype Ontology (APO), but FYPO includes more detailed pre-composed terms as well as computable definitions.
domain: phenotype
homepage: https://sourceforge.net/apps/trac/pombase/wiki/FissionYeastPhenotypeOntology
id: fypo
products: 
  - id: fypo.owl
taxon: 
  id: NCBITaxon:4896
  label: S Pombe
title: Fission Yeast Phenotype Ontology
tracker: http://sourceforge.net/tracker/?group_id=65526&atid=2096431
contact: 
  email: jaiswalp@science.oregonstate.edu
  label: Pankaj Jaiswal
description: A structured, controlled vocabulary for the representation of plant environmental conditions.
domain: environment
homepage: http://www.gramene.org
id: eo
page: http://www.gramene.org/plant_ontology/index.html#eo
products: 
  - id: eo.owl
title: Plant Environmental Conditions
contact: 
  email: shimoyama@mcw.edu
  label: Mary Shimoyama
description: The Experimental Conditions Ontology is designed to represent the conditions under which physiological and morphological measurements are made both in the clinic and in studies involving humans or model organisms.
domain: clinical
id: xco
products: 
  - id: xco.owl
title: Experimental condition ontology
contact: 
  email: albertgoldfain@gmail.com
  label: Albert Goldfain
description: The Ontology for General Medical Science (OGMS) is based on the papers Toward an Ontological Treatment of Disease and Diagnosis and On Carcinomas and Other Pathological Entities. The ontology attempts to address some of the issues raised at the Workshop on Ontology of Diseases (Dallas, TX) and the Signs, Symptoms, and Findings Workshop(Milan, Italy). OGMS was formerly called the clinical phenotype ontology. Terms from OGMS hang from the Basic Formal Ontology. See http://ontology.buffalo.edu/medo/Disease_and_Diagnosis.pdf
domain: medicine
homepage: http://code.google.com/p/ogms/
id: ogms
page: http://code.google.com/p/ogms/
products: 
  - id: ogms.owl
taxon: 
  id: NCBITaxon:9606
  label: Homo sapiens
title: Ontology for General Medical Science
tracker: http://code.google.com/p/ogms/issues/list
---

The Ontology for General Medical Science (OGMS) is based on the papers Toward an Ontological Treatment of Disease and Diagnosis and On Carcinomas and Other Pathological Entities. The ontology attempts to address some of the issues raised at the Workshop on Ontology of Diseases (Dallas, TX) and the Signs, Symptoms, and Findings Workshop(Milan, Italy). OGMS was formerly called the clinical phenotype ontology. Terms from OGMS hang from the Basic Formal Ontology. See http://ontology.buffalo.edu/medo/Disease_and_Diagnosis.pdf